opts/ip_test.go

```diff
@@ -48,7 +50,7 @@ func TestIpOptSetInvalidVal(t *testing.T) {
 	invalidIP := "invalid ip"
 	expectedError := "invalid ip is not an ip address"
```

Comment:
This is not as simple, by design derr.ErrorCodeInvalidIPFormat.Message gives you raw message, we will have to call the function with args.
The best is to go through phase II to eliminate static strings.